guides/source/configuring.md

```diff
@@ -256,11 +256,23 @@ Guide
 
 #### `config.credentials.content_path`
 
-Configures lookup path for encrypted credentials.
+The path of the encrypted credentials file.
+
+Defaults to `config/credentials/#{Rails.env}.yml.enc` if it exists, or
+`config/credentials.yml.enc` otherwise.
+
+NOTE: In order for the `bin/rails credentials` commands to recognize this value,
```

Comment:
> I have a follow-up commit for supporting `config.credentials.content_path` and `config.credentials.key_path` in `config/environments/*.rb`.

Is this a big change? If it's not, I think it's worth including in this PR, so that this warning isn't required.

Reply:
It is based on top of a refactor, which is also a separate commit.  Additionally, it involves running some but not all initializers, which is a larger change in behavior.  I think it will better as a separate PR.

And, even with that PR, the warning will still be required because it will only add support for `config/environments/*.rb`, not `config/initializers/*.rb`.